### PR TITLE
Update training interface and removing info on invoice to( train-ticket)

### PR DESCRIPTION
**Pull request by fakela**
This PR is in relation with this issue #13

### CODE REVIEW

File: build-your-first-document-parsing-api/training-interface.md [hunk 1]
```diff
@@ -46,22 +68,39 @@ You land now on the setup page. Here is the [invoice](https://mindee-public-webs
...
+}
+[/block]
 
-To skip past the setup of the API, we'll use the JSON upload to create the API.  Upload [this json file](https://mindee-public-website.s3.amazonaws.com/blog/2021/04/25/how_it_works-config1.json) into the data model. Once it’s done, it should looks like this:
+Alternatively, you can build the data model manually:
 
- 
+- **Passenger name**: String type that never contains numeric characters.
+- **Reservation number**: String type without specifications.
+- **Departure Date**: Date type with US format.
```
Comment (fharper) [inline build-your-first-document-parsing-api/training-interface.md:89]: with US format (MM-DD-YYYY)

for people that wouldn't be familiar with it

**Review by fharper — CHANGES_REQUESTED** [review]
Just the small addition, and you can merge it

File: build-your-first-document-parsing-api/training-interface.md (hunk 1 above)
Comment (fakela) [reply build-your-first-document-parsing-api/training-interface.md]: Done 👍

**Review by fharper — APPROVED** [review]
LGTM

**Comment by fharper** [comment]
I'll let you squash and merge with a better commit message @fakela unless you think all commits make sense, in that case, just rebase and merge :)

**Pull request merged**